### PR TITLE
Add 404

### PR DESCRIPTION
**Pull request by ocefpaf**
@kbailey-noaa and @MathewBiddle this PR removes all the pages in the gh-pages here, sets a 404 page, and a redirect to the new one for the landing page. The redirect is set only to the main landing page, so folks with a direct link to a sub-page will get the 404 instead.

PS: This whole branch, with its history, is available in the https://ioos.github.io/glider-dac gh-pages now.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+</head>
+
+<body>
+	<h1 style="text-align:center;color:rgb(19, 0, 128);">
+		IOOS CodeLab
+	</h1>
+	
+	<p style="text-align:center;">
+		If your browser supports Refresh,
+		you'll be redirected to IOOS CodeLab
```
[the remark this answers is not in the document]
Comment (ocefpaf) [reply index.html]: Ooops. Fixed that in the next commit.

[the remark this answers is not in the document]
Comment (MathewBiddle) [reply index.html]: thx!

**Comment by MathewBiddle** [comment]
LGTM. @kbailey-noaa should have final say before we merge this in.

**Review by kbailey-noaa — CHANGES_REQUESTED** [review]
index.html changes -- the <body> section mentions IOOD CodeLab.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+<head>
+	<title>HTML Redirect</title>
+	<meta http-equiv="refresh"
+		content="5; url = https://ioos.github.io/glider-dac" />
+</head>
+
+<body>
+	<h1 style="text-align:center;color:rgb(19, 0, 128);">
+		IOOS CodeLab
```
Comment (MathewBiddle) [inline index.html:12]: IOOS Glider DAC

**Comment by ocefpaf** [comment]
> index.html changes -- the section mentions IOOS CodeLab.

Should be OK in the last commit.

**Comment by MathewBiddle** [comment]
LGTM! Thanks @ocefpaf!

**Review by kbailey-noaa — APPROVED** [review]

**Pull request merged**